android/CouchbaseLite/src/ce/java/com/couchbase/lite/ReplicatorConfiguration.java

```diff
@@ -284,6 +316,18 @@ public Authenticator getAuthenticator() {
         return documentIDs;
     }
 
+    /**
+     * Gets a filter closure for validating whether the documents can be pushed
```

Comment:
On Java platform, the ReplicationFilter is not a closure. I think use filter object is better.